src/pages/PremiumVideoCall/state/usePasscodeAuth/usePasscodeAuth.ts

```diff
@@ -87,7 +109,9 @@ export default function usePasscodeAuth() {
                         history.replace(window.location.pathname)
                     }
                 })
-                .then(() => setIsAuthReady(true))
```

Comment:
async/await ?

Reply:
this code is from the original [Twilio Video App project](https://github.com/twilio/twilio-video-app-react). I think we'll have to state in the future if to update all that code in order to meet our code styling guidelines

Reply:
Ok